demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -1044,17 +1047,15 @@ def validate_modeling_rule(
                 )
                 logger.error(
                     f"[red]Please create a test data file for "
-                    f"{get_relative_path_to_content(modeling_rule_directory)} and then rerun[/red]",
+                    f"{get_relative_path_to_content(modeling_rule_directory)} and then rerun\n{executed_command}[/red]",
                     extra={"markup": True},
                 )
-                typer.echo(executed_command)
         else:
             logger.error(
                 f"[red]Please create a test data file for "
-                f"{get_relative_path_to_content(modeling_rule_directory)} and then rerun[/red]",
+                f"{get_relative_path_to_content(modeling_rule_directory)} and then rerun\n{executed_command}[/red]",
```

Comment:
Same comment.
Also: Can the messages be unique?

Reply:
will fix